Source/ConvolutionFunctions/arm_depthwise_conv_get_buffer_sizes_s16.c

```diff
@@ -32,11 +32,11 @@
 #include "arm_nnsupportfunctions.h"
 
 /**
- *  @ingroup Public
+ *  @ingroup NNconv
  */
 
 /**
- * @addtogroup NNConv
+ * @addtogroup InternalNNConv
```

Comment:
So, part of these are internal while the rest are still Public API(e.g arm_convolve_wrapper_s8_get_buffer_size) and we need to make that differentiation so that internal API's aren't mentioned in the Public section. 

So, I see something like this..
Public
     -----NNConv
                ----------GetBufferSize
     -----FullyConnected
                 -----------GetBufferSize

Private
     ------Conv
               ------GetBufferSize

and so on..

Reply:
We can use the same sub group name(GetBufferSize) for in all groups(Conv, FC , Pooling..)..both Public and Private..

Reply:
Change to GetBufferSize<ParentGroup>. If using just GetBufferSize all of the functions are mixed up